composer.json

```diff
@@ -17,11 +17,17 @@
         "dhii/php-cs-fixer-config": "dev-php-5.3",
         "phpunit/phpunit": "^4.8",
         "ptrofimov/xpmock": "^1.1",
-        "codeclimate/php-test-reporter": "<=0.3.2"
+        "codeclimate/php-test-reporter": "<=0.3.2",
+        "dhii/validation-interface": "^0.1"
```

Comment:
## Summary of Voice Call

The `try ... catch` works the same as `instanceof` operator, in the sense that if the class/interface foes not exist, a fatal error is not thrown.

Therefore the `dhii/validation-interface` is not a requirement. It has been agreed that the package will however be listed as a Composer suggested package.